framework/src/modules/interoperability/mainchain/commands/register_sidechain.ts

```diff
@@ -71,9 +72,7 @@ export class RegisterSidechainCommand extends BaseInteroperabilityCommand<Mainch
 		if (!isValidName(name)) {
 			return {
 				status: VerifyStatus.FAIL,
-				error: new Error(
-					`Invalid name property. It should contain only characters from the set [a-z0-9!@$&_.].`,
-				),
+				error: new Error(getInvalidNameErrMsg()),
```

Comment:
Instead of having a function to get the constant string, it would be better to either
* Define it as a constant
* Define new error type with this message